test/unit/org/apache/cassandra/index/sai/cql/AllowFilteringTest.java

```diff
@@ -296,6 +296,79 @@ public void testAllowFilteringOnClusteringAndRegularColumns() throws Throwable
         test("SELECT * FROM %s WHERE v1=0 AND v2=0 AND k1=0 AND k2=0 AND (c1, c2, c3, c4) = (0, 0, 0, 0) AND v3=0", true);
     }
 
+    @Test
+    public void testAllowFilteringTextWithINClause () throws Throwable
```

Comment:
nit: unnecessary `throws Throwable` here and a few places below

Reply:
Removed `throws Throwable.`